content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
+weight: 46
+keywords: [external,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-istiod/
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
+
+## Introduction
+
+The {{< gloss >}}external control plane{{< /gloss >}} [deployment model](/docs/ops/deployment/deployment-models/#control-plane-models) enables mesh operators to install and manage mesh control planes on separate external clusters. This deployment model allows a clear separation between mesh operators and mesh admins. Istio mesh operators can run Istio control planes for mesh admins while mesh admins control the configuration of the control plane without worrying about installing or managing it.
+
+## Requirements
+
+### Clusters
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
+
+The first cluster is the {{< gloss >}}external control plane{{< /gloss >}} cluster with the Istio `default` profile installed in the `external-istiod` namespace. This istiod serves as the the external control plane for the second cluster.  An ingress gateway is installed in the `istio-system` namespace. The ingress gateway provides mesh sidecars access to the external istiod in the `external-istiod` namespace.
+
+The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} hosting the mesh. Its Kubernetes API server also provides the configuration for the control plane (istiod) running in the external cluster.
+
+### API Server Access
+
+The API Server in the remote config cluster must be accessible to the external control plane cluster. Many cloud providers make API Servers publicly accessible via network
+load balancers (NLBs). If the API Server is not directly accessible, you will
+have to modify the installation procedure to enable access. For example, the
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateway used in
+the multi-network and primary-remote configurations could also be used
+to enable access to the API Server.
+
+## Environment Variables
+
+This guide will refer to two clusters named `external_cluster` and `remote_cluster`. The following environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_EXTERNAL_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the external control plane cluster.
```

Comment:
Let's just use these for brevity: `CTX_EXTERNAL`, `CTX_REMOTE`, `ISTIOD_ADDR`, `SECRET_NAME`

Reply:
Can we leave out these changes till we have more time?  Trying to get this PR in first so we can claim external istiod alpha before the release.

Reply:
I know this is pre-alpha, but readability is still important. I think trimming these down should be a relatively simple change, no?